Clinical Key.js

```diff
@@ -130,6 +140,117 @@ function doWeb(doc, url){
 		search.setSearch({ ISBN: isbn });
 		search.getTranslators();
 	}
+	else if (contentType == 'journalArticle') {
+		eid = url.split('/');
+		pii = eid.pop().slice(7);
+		
+		/* can use xpath to get pii; guessing the url is faster
+		var eid = ZU.xpathText(doc, "//ul/@data-eid");
+		pii = eid.slice(7);*/
+		
+		if(!/^S(\d{15}X|\d{16})/.test(pii)){
+			throw new Error('PII incorrectly formatted');
+			return;
+		}
+		
+		doSearch(pii, doc);
+	}
+	
+}
+
+//Search & Processing based on CrossRef.js translator
+function doSearch(pii, doc){
+	crossRefQuery = 'http://api.crossref.org/works?query=' + pii
+	Zotero.Utilities.HTTP.doGet(crossRefQuery, function(responseText) {
```

Comment:
In my comment below, I mentioned attempting to specify the API version using an Accept Header in the HTTP request. Currently, that doesn't seem to work. However, if the problem is solved, based on the criteria you describe in #1242, would it be appropriate then to continue using the ZU.HTTP.doGet?